advanced_source/cpp_custom_ops.rst

```diff
@@ -62,41 +62,30 @@ Using ``cpp_extension`` is as simple as writing the following ``setup.py``:
 
   setup(name="extension_cpp",
         ext_modules=[
-            cpp_extension.CppExtension("extension_cpp", ["muladd.cpp"])],
-        cmdclass={'build_ext': cpp_extension.BuildExtension})
+            cpp_extension.CppExtension(
+              "extension_cpp",
+              ["muladd.cpp"]
+              py_limited_api=True)],
+        cmdclass={'build_ext': cpp_extension.BuildExtension},
+        options={"bdist_wheel": {"py_limited_api": "cp39"}}
```

Comment:
What does `{"py_limited_api": "cp39"}` mean? That 3.9 is the smallest version we support?

Reply:
Ah I see it's metnioned below

Reply:
yeh

Reply:
i can also add a comment